src/Components/addItem.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// import ReactDOM from 'react-dom';
+import { fb } from '../lib/firebase';
+// import { FirestoreProvider } from 'react-firestore';
```

Comment:
Woops, there are unused imports here - They're commented out, that's why the linter is not complaining about this!

You'd like to remove this as well before merging!